pyquil/experiment/_setting.py

```diff
@@ -192,8 +192,14 @@ class ExperimentSetting:
 
     in_state: TensorProductState
     out_operator: PauliTerm
-
-    def __init__(self, in_state: TensorProductState, out_operator: PauliTerm):
+    additional_expectations: Optional[List[List[int]]] = None
```

Comment:
Why is this `List[List[int]]` rather than something like `List[PauliTerm]`? I'm ok with either, but it seems odd/inconsistent with the typical usage of `ExperimentSetting`. It's worth documenting this (even as a comment) I think